tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -16,6 +16,19 @@ function goHomeJoinNewGame() {
   cy.log('Joined new game successfully');
 }
 
+function validateMatch(match, length, p1, p2) {
```

Comment:
For consistency maybe this could be
```suggestion
function validateMatchResult(match, length, p1, p2) {
```